packages/wdio-cli/src/constants.ts

```diff
@@ -431,8 +432,18 @@ export const QUESTIONNAIRE = [{
     // @ts-ignore
     default: [SUPPORTED_PACKAGES.service.find(
         /* istanbul ignore next */
-        ({ name }) => name === 'chromedriver').value
-    ],
+        ({ name }) => name === 'browserstack').value],
+    when: /* istanbul ignore next */ (answers: Questionnair) => answers.backend === 'In the cloud using Browserstack',
+    validate: /* istanbul ignore next */ (answers: string[]) => validateServiceAnswers(answers)
+}, {
+    type: 'checkbox',
+    name: 'services',
```

Comment:
I suggest a more explicit name
```suggestion
    name: 'addBrowserStackService',
```

Reply:
please see comment below, let's keep a single entry for selecting services.